nc_news/api.js

```diff
@@ -7,5 +7,21 @@ const getArticles = () => {
       return data;
     });
 };
+const getArticleById = (article_id) => {
+  return axios
+    .get(`https://nc-news-mr7q.onrender.com/api/articles/${article_id}`)
+    .then((data) => {
+      return data.data.article;
+    });
+};
 
-export { getArticles };
+const getAllComments = (article_id) => {
```

Comment:
I would maybe call this getCommentsById (just so it is more clear what it is fetching), as it isn't all comments for every article) 🙂